src/mds/topology/topology_service_manager.cpp

```diff
@@ -751,11 +751,72 @@ void TopologyServiceManager::ListPoolZone(const ListPoolZoneRequest* request,
     }
 }
 
+void TopologyServiceManager::CreatePoolset(
+    const PoolsetRequest *request,
+    PoolsetResponse *response) {
+    if ((request->has_poolsetname()) && (request->has_type()) &&
+        (request->has_desc())) {
+        PoolIdType pid = topology_->AllocatePoolsetId();
```

Comment:
Why `AllocatePoolsetId` returns `PoolIdType` instead of `PoolsetIdType`?

Reply:
mistake in coding , fix

> Why `AllocatePoolsetId` returns `PoolIdType` instead of `PoolsetIdType`?

mistake in coding, fix

Reply:
> Why `AllocatePoolsetId` returns `PoolIdType` instead of `PoolsetIdType`?

mistake in coding, fix